packages/ui/src/components/Header/Header.tsx

```diff
@@ -1,33 +1,58 @@
+'use client'
 import { Icon } from '../Icon'
 import { twMerge } from 'tailwind-merge'
 
 export interface HeaderProps {
+  page?: string
```

Comment:
In real-world projects, you won't be getting the page name through props. Instead, you need to fetch it from the page url through usePathname() incase of next js. please use that and make it work.